src/Symfony/Component/Cache/Tests/Adapter/TagAwareAdapterTest.php

```diff
@@ -71,6 +71,22 @@ public function testInvalidateTags()
         $this->assertTrue($pool->getItem('foo')->isHit());
     }
 
+    public function testInvalidateCommits()
+    {
+        $pool1 = $this->createCachePool();
+
+        $foo = $pool1->getItem('foo');
+        $foo->tag('tag');
+
+        $pool1->saveDeferred($foo->set('foo'));
+        $pool1->invalidateTags(array('foo'));
```

Comment:
I'm confused, `'foo'` should be `'tag'` here, right?

Reply:
good catch! fixed